eahub/localgroups/forms.py

```diff
@@ -21,12 +22,11 @@ def __init__(self, *, user, local_group, **kwargs):
         queryset = base_models.User.objects.select_related("profile").annotate(
             already_selected=already_selected
         )
-        if not user.is_superuser:
-            queryset = queryset.filter(
-                models.Q(profile__is_public=True)
-                | models.Q(already_selected=True)
-                | models.Q(pk=user.pk)
-            )
+        queryset = queryset.filter(
+            models.Q(profile__is_public=True, profile__is_approved=True)
+            | models.Q(already_selected=True)
+            | models.Q(pk=user.pk)
+        )
```

Comment:
I wasn't sure why "superuser" here is important, hence I removed it for simplicity sake, otherwise it's hard to test since I'm seeing bots in the dropdown as an admin. I recommend to do anything admin related in the admin panel.